src/OlStyleParser.ts

```diff
@@ -89,6 +89,30 @@ class OlStyleParser implements StyleParser {
     };
   }
 
+  /**
+   * Get the GeoStyler-Style TextSymbolizer from an OpenLayers Style object.
+   *
+   *
+   * @param {ol.style.Style} olStyle The OpenLayers Style object
+   * @return {TextSymbolizer} The GeoStyler-Style TextSymbolizer
+   */
+  getTextSymbolizerFromOlStyle(olStyle: ol.style.Style): TextSymbolizer {
+    const olTextStyle = olStyle.getText() as ol.style.Text;
+    const olFillStyle = olTextStyle.getFill() as ol.style.Fill;
+    const offsetX = olTextStyle.getOffsetX();
+    const offsetY = olTextStyle.getOffsetY();
+    const font = olTextStyle.getFont();
+    const fontSize = parseInt(font.split('px')[0], 10);
```

Comment:
Since font in an `ol.style.Text` is a CSS 'font' value this does not seem to work for all cases. E. g. when a font-weight is set: `bold 5px arial, sans-serif`
Even more complicated cases are thinkable: `italic bold 12px/30px Georgia, serif`.

Reply:
In c58544eaf6 I added some logic to parse even more complex CSS font values.

Reply:
However, this currently only works with font-size values given in pixel.
Since font-size can be a string value as well (see https://www.w3schools.com/cssref/pr_font_font-size.asp, e.g. `large`) the model of `TextSymbolizer` (`size?: number;`) should be changed. This is is beyond the scope of this PR, IMHO. What do you think @chrismayer @KaiVolland ?

Reply:
I don't think we need to support this atm. This would make many things very complicated.

But we should improve the feedback of the parser. Maybe return an feedback object or throw errors. I created a ticket regarding this: https://github.com/terrestris/geostyler/issues/286